docker/yb-voyager-docker

```diff
@@ -1,26 +1,39 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This script has inconsistent spacing. Some lines use tabs and other lines use white-spaces for indentation. Please use tabs everywhere for indentation.

Reply:
It was actually using all tabs. There were extra tabs where I removed the if conditions😅